source/lesson-2-ruby-and-the-command-line.html.md.erb

```diff
@@ -0,0 +1,722 @@
+
+Once you opened up your terminal you should see a window like this:
+
+![Screenshot of the macOs terminal app. The text says Last login: Sat May 18 11:08:55 on ttys002 ~$ ](/images/lesson-2/image1.png)
+
+Don't worry if the text in yours is a little different, it doesn't matter.
+
+In our examples, we'll write `$` to represent the prompt, you don't have to
+type this just the words after, followed by enter to run the command.
+
+Try running:
+
+```shell
+$ pwd
+```
+
+#### `pwd` or print working directory
+
+The `pwd` command prints out the current directory (another name for folder)
+you are in. If you just opened up your terminal, you are probably in your
+"home" directory, and should get an output similar to this:
+
+```
+/Users/your-username
+```
+
+So your current "working directory" is `/Users/your-username`.
+
+#### Getting things wrong on the command line
+
+If you type a command that the command line doesn't understand, it will show
+you an error message. Don't panic, if you see one of these - everything is fine!
+Just have a look at the command you wrote and see if you can work out what was wrong.
+
+Try this for example:
+
+```shell
+$ whargleblargle
+```
+
+You should see an error message like `-bash: whargleblargle: command not found`.
+
+#### `ls` or list
+
+You might wonder "how I do know which files are in a directory?", the `ls` command can do this:
+
+```shell
+$ ls
+```
+
+This should print a list of the files and folders inside the working
+directory. You'll probably see directories like `Applications`, `Desktop`,
+`Documents` and `Downloads`.
+
+### `cd` or change directory
+
+The `cd` command allows you to move between directories. You tell `cd` which directory to move to by putting the path after the `cd`, like this:
+
+```shell
+$ cd 'Desktop'
+```
+
+Lots of commands need parameters like this (e.g. `cd` needs to know the directory to move to, while `pwd` does not). We call the parameters "arguments".
+
+### Task 1: change to the directory containing your code
+
+In Lesson 1 we created a folder to keep our source code
+(`lesson-1-html-and-css`). Create a new folder for lesson 2 called
+`lesson-2-ruby` and change the working directory in your command line using
+`cd`.
+
+> **Notes**
+
+> In macOs Finder you can copy the path to a directory by holding down
+the option key (`⌥`), right clicking, and choosing *Copy
+"Directory" as Pathname*.
+
+> If your directory includes spaces (or other weird characters) you might
+need to put it in quotes so the command line doesn't get confused. So:
+
+> `$ cd '/Users/your-username/My Directory With Spaces/lesson-2-ruby'`
+
+> instead of
+
+> `$ cd /Users/your-username/My Directory With Spaces/lesson-2-ruby`
+
+Confirm you're in the right place by running:
+
+```shell
+$ pwd
+```
+
+and check what files there are in this directory by running:
+
+```shell
+$ ls
+```
+
+There probably won't be any files yet, because we haven't created any.
+
+## Introduction to ruby
```

Comment:
Mixture of capitalised `Ruby` and non-capitalised `ruby` in the doc - can we stick with one (capitalised)?

Reply:
Capitalised everywhere now 🎉